Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -171,7 +173,21 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
   switch (req.command)
   {
   case GO_IDLE_STATE:
-    // libogc can use it during init..
+    INFO_LOG(IOS_SD, "GO_IDLE_STATE");
+    if (m_Card)
+    {
+      if (m_Card.GetSize() > SDHC_BYTES)
+      {
+        m_Status |= CARD_SDHC;
+        Memory::Write_U32(0xc0ff8000, _BufferOut);
```

Comment:
That mask looks awfully familiar (to the OCR further down). Can you just pass in `m_Status` here instead? Or maybe some other member that reflects the value to be sent here (in case there should be one that keeps the OCR or whatever over time)?